### PR TITLE
Fix for import

### PR DESCRIPTION
**Pull request by derjust**
Fix for https://wordpress.org/support/topic/0-byte-files/

### CODE REVIEW

**Comment by allan23** [comment]
Hi @derjust,
Thank you for the pull request. We are reviewing and will advise accordingly.

**Comment by allan23** [comment]
Closing in favor of #19, due to delay. However, thank you for your pull request.

File: windows-azure-storage.php [hunk 1]
```diff
@@ -383,11 +391,14 @@ function windows_azure_storage_wp_update_attachment_metadata( $data, $post_id )
 	if ( '/' === $upload_dir['subdir']{0} ) {
 		$upload_dir['subdir'] = substr( $upload_dir['subdir'], 1 );
 	}
-	
+
 	// Prepare blob name.
 	$relative_file_name = ( '' === $upload_dir['subdir'] ) ?
-		basename( $upload_file_name ) :
-		$upload_dir['subdir'] . '/' . basename( $upload_file_name );
+		basename( $upload_file_name ) : //TODO SJ I think this is also covered by the next line
```
Comment (derjust) [inline windows-azure-storage.php]: Leftover. Wasn't meant to be included.

**Comment by derjust** [comment]
No worries. Thanks for the plugin!

**Pull request closed without merging**